docs/docs/model-storage.mdx

```diff
@@ -2,156 +2,214 @@
 
-Rasa tries to load a model in the above mentioned order, i.e. it only tries to load your model from your local
-storage system if no model server and no remote storage were configured.
+Loading the model from your local disk will be used as a fallback if you
+don't specify how a model should be loaded.
 
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="server-fetch-from-server"></a>
+## Load Model from Disk
 
-## Fetching Models from a Server
+By default models will be loaded from your local disk. You can specify the path
+to your model with the `--model` parameter:
+
+```bash
+rasa run --model models/20190506-100418.tar.gz
+```
+
+If you want to load the latest model in a directory, you can specify
+a directory instead of a file:
+
+```bash
+rasa run --model models/
+```
+
+Rasa will check all the models in that directory and load the one which was trained
+most recently.
+
+Rasa will assume `models/` to be the default model directory and
+will use it in case no `--model` argument is specified. The two following calls
+will load the same model:
+
+```bash
+# this command will load the same model
+rasa run --model models/
+# ... as this command (using defaults)
+rasa run
+```
+
+## Load Model from Server
+
+You can configure the Rasa Open Source server to regularly fetch
+a model from a server and deploy it.
+
+#### Configuration of Rasa Open Source
 
 You can configure the HTTP server to fetch models from another URL
-by adding it to your `endpoints.yml`. The server will then
-reguarly query this URL for zipped models.
+by adding it to your `endpoints.yml`:
 
 ```yaml-rasa title="endpoints.yml"
 models:
-  url: http://my-server.com/models/default@latest
+  url: http://my-server.com/models/default
   wait_time_between_pulls: 10   # In seconds, optional, default: 100
 ```
+The server will query `url` for a zipped model every `wait_time_between_pulls`
+seconds.
 
-:::note
-If you want to pull the model just once from the server, set
-`wait_time_between_pulls` to `None`.
-
-:::
-
-:::note
-Your model server must provide zipped Rasa models, and have
-`{"ETag": <model_hash_string>}` as one of its headers. Rasa will
-only download a new model if this model hash has changed.
+If you want to pull the model just once from the server, you can set the time
+between pulls to `null`:
 
-:::
+```yaml-rasa title="endpoints.yml"
+models:
+  url: http://my-server.com/models/default
+  wait_time_between_pulls: null  # fetches model only once
+```
 
-Rasa sends requests to your model server with an `If-None-Match`
-header that contains the current model hash. If your model server can
-provide a model with a different hash from the one you sent, it should send it
-in as a zip file with an `ETag` header containing the new hash. If not, Rasa
-expects an empty response with a `204` or `304` status code.
+#### Configuration of your server
```

Comment:
```suggestion
#### Configuration of Your Server
```

Reply:
what about "How to Configure Your Server" ? would be more straightforward i think